dependencies/supported_version/Trilinos.txt

```diff
@@ -1,3 +1,4 @@
+ef517e44826ce3a9e0189325aec4eee6e1d4dafe 2025.5 none
 06db4c850654feacabdaed61ee8308219266b6a5 2025.4 none
 8cdf0375d278598c12edc3fd7059cb133c6f4426 2025.3 16.2.0
```

Comment:
Not part of this PR, but anyways: 16.2.0 has not been released, yet. The most recent Trilinos release is 16.1.0. See https://github.com/trilinos/Trilinos/releases

Reply:
That's true, I will adjust this on judgement day in November ;).